otter/metrics.py

```diff
@@ -260,18 +277,25 @@ def collect_metrics(reactor, config, client=None, authenticator=None,
 
     :return: :class:`Deferred` with None
     """
+    convergence_tids = None
+    if config.get('convergence-tenants', []):
+        convergence_tids = config['convergence-tenants']
     _client = client or connect_cass_servers(reactor, config['cassandra'])
     authenticator = authenticator or generate_authenticator(reactor,
                                                             config['identity'])
     service_configs = get_service_configs(config)
 
-    dispatcher = get_full_dispatcher(reactor, authenticator, metrics_log,
-                                     service_configs)
+    dispatcher = get_legacy_dispatcher(reactor, authenticator, metrics_log,
+                                       service_configs)
 
     # calculate metrics
-    cass_groups = yield get_scaling_groups(
-        _client, props=['status'],
-        group_pred=lambda g: g['status'] != 'DISABLED')
+    if convergence_tids:
```

Comment:
Actually, the way I did it was an intentional choice, since I didn't want to run this codepath in the case of an empty list of convergence tenants.

Mostly this was because I was assuming that "SELECT * WHERE FOO IN ()" would be a syntax error, as it in is Postgres. But apparently in CQL it's not a syntax error, so I guess I can change it :)